### DISCUSSION TITLE
Existing models in Automatic1111

### DISCUSSION THREAD

**Discussion by wisdombox44** [lllyasviel/stable-diffusion-webui-forge#193] [General]
Hi Thank you for great work,
I have few questions
1. How can I work with existing models stored in Automatic1111 without moving them in to Forge
2. How Can I create a default preset for Turbo
3. How can I enable dark theme
4. How can copy Fooocus styles.
Thank you

**Comment by kuzivas** [comment]
in the webui-user.bat in the line of arguments       set COMMANDLINE_ARGS=     I wrote the path to my folders with models and LoRAs

```
@echo off

set PYTHON=
set GIT=
set VENV_DIR=
set COMMANDLINE_ARGS= --ckpt-dir "F:\SD1.6.RC\stable-diffusion-webui\models\Stable-diffusion" --lora-dir "F:\SD1.6.RC\stable-diffusion-webui\models\Lora"
@REM Uncomment following code to reference an existing A1111 checkout.
@REM set A1111_HOME=Your A1111 checkout dir
@REM
@REM set VENV_DIR=%A1111_HOME%/venv
@REM set COMMANDLINE_ARGS=%COMMANDLINE_ARGS%^
@REM  --ckpt-dir %A1111_HOME%/models/Stable-diffusion^
@REM  --hypernetwork-dir %A1111_HOME%/models/hypernetworks^
@REM  --embeddings-dir %A1111_HOME%/models/embeddings^
@REM  --lora-dir %A1111_HOME%/models/Lora

call webui.bat
```

**Reply by wisdombox44** [reply]
Thank you for your quick reply 👍

**Comment by huchenlei** [comment]
After https://github.com/lllyasviel/stable-diffusion-webui-forge/pull/203, you can use --forge-ref-a1111-home commandline arg to reference models in existing A1111 checkout.

**Reply by xlinx** [reply]
example. edit
"C:\Users\x\_ML\webui_forge_cu121_torch21\webui\webui.bat"
remember add path after {--forge-ref-a1111-home}
`@echo off

set PYTHON=
set GIT=
set VENV_DIR=
set COMMANDLINE_ARGS=--forge-ref-a1111-home C:\\Users\\x\\_ML\\stable-diffusion-webui

@REM Uncomment following code to reference an existing A1111 checkout.
@REM set A1111_HOME=C:\\Users\\x\\_ML\\stable-diffusion-webui
@REM set VENV_DIR=%A1111_HOME%/venv
@REM set COMMANDLINE_ARGS=%COMMANDLINE_ARGS% --ckpt-dir %A1111_HOME%/models/Stable-diffusion --vae-dir
@REM %A1111_HOME%/models/VAE --hypernetwork-dir %A1111_HOME%/models/hypernetworks --embeddings-dir 
@REM %A1111_HOME%/models/embeddings --lora-dir %A1111_HOME%/models/Lora

call webui.bat`

**Comment by Rifz42** [comment]
thanks kuzivas!
the @Rem examples aren't helpful because they don't show the ""  here's what worked for me, plus Dark mode.
set COMMANDLINE_ARGS= --ckpt-dir "C:/Ai1111/stable-diffusion-webui/models/Stable-diffusion" --lora-dir "C:/Ai1111/stable-diffusion-webui/models/Lora"

call webui.bat --theme dark

**Comment by xlinx** [comment]
```
@echo off

set PYTHON=
set GIT=
set VENV_DIR=
set COMMANDLINE_ARGS=

@REM Uncomment following code to reference an existing A1111 checkout.
set A1111_HOME=C:\\Users\\x\\_ML\\stable-diffusion-webui

set VENV_DIR=%A1111_HOME%/venv
set COMMANDLINE_ARGS=%COMMANDLINE_ARGS% --ckpt-dir %A1111_HOME%/models/Stable-diffusion --vae-dir %A1111_HOME%/models/VAE --hypernetwork-dir %A1111_HOME%/models/hypernetworks --embeddings-dir %A1111_HOME%/models/embeddings --lora-dir %A1111_HOME%/models/Lora

call webui.bat
```

**Comment by xlinx** [comment]
i'm trying also link extensions
must use hard link by command ; DONTuse mouse right click shortlink

> mklink /D C:\Users\x\_ML\webui_forge_cu121_torch21\webui\extensions C:\Users\x\_ML\stable-diffusion-webui\extensions
